### ISSUE TITLE
Missed shows changing from Wanted to Skipped

### ISSUE THREAD

**Issue by Yellow1144** [SickRage/SickRage#3592]
Branch/Commit: Master/a1f5e6750493b00b032536c9a4274da57b5b9531
OS:Win7Pro/64
What you did:
What happened: I noticed recently that missed episodes from various shows were not being downloaded. I looked and noticed that their status had been changed to Skipped instead of staying at Wanted until they are snatched. I'm not sure exactly when this changed. Some shows might not be available for days or weeks after their airdate. It sucks to have to manually reset these back to wanted which was the way SR operated up til now. 

I can find no option anywhere to Change this behavior. A solution would not be to change Missed Episode Range to a much later time as I don't want to see the missed episodes stay in the schedule beyond a week. They should just fall off the schedule and stay wanted as they used to. 

Thank You

**Comment by sharkykh** [comment]
You can change this by editing the show's configuration.
There is a setting named Default Episode Status.
You can also change this and set the defaults when you add a new show or an existing show.

**Comment by Yellow1144** [comment]
No, this is happening in shows whose default status is already set to wanted. On the day the episode airs, the episode of the day becomes wanted. If it isn't snatched right away, it becomes "Missed" in the schedule for the 7 days I have it set in Missed Episode Range. After that it is getting set to Skipped instead of remaining as Wanted until it is snatched what could be months later.  I have the status set to wanted as my default whenever adding a show. This is happening with shows that I have had in SR for a couple of seasons. I have been using SR since the beginning, the old repo and this behavior just started recently.

**Comment by sharkykh** [comment]
@Yellow1144 
Sorry it took me a while to realize.
It sounds a bit like a coincidence, but it just might be the case:
What is the value you have set for `Backlog search day(s)` in http://localhost:8081/config/search/#episode-search?

**Comment by Yellow1144** [comment]
It is set to 14 days, as I have always had it set. 
At the moment I do not have any shows that are missing episodes longer than 14 days that have not either downloaded normally, I have manually downloaded or I have previously manually set to wanted, thus at the moment I have no new instances of wanted shows being set to skipped. So I have no conditions currently to test this.

**Comment by sharkykh** [comment]
@Yellow1144 
Try enabling debug logging,
then the next time you have a missed show that turned to skipped
check the debug logs (in Log Viewer),
there's usually some information about operations like that.
